assets/queries/crossplane/aws/sqs_with_sse_disabled/metadata.json

```
@@ -0,0 +1,11 @@
+{
+  "id": "9296f1cc-7a40-45de-bd41-f31745488a0e",
+  "queryName": "SQS with SSE disabled",
+  "severity": "MEDIUM",
+  "category": "Encryption",
+  "descriptionText": "Amazon Simple Queue Service (SQS) queue is not protecting the contents of their messages using Server-Side Encryption (SSE)",
```

```suggestion
"descriptionText": "Amazon Simple Queue Service (SQS) queue should protect the contents of their messages using Server-Side Encryption (SSE)",
```

Please, update all the other related queries